.github/workflows/localization-automerge.yml

```diff
@@ -0,0 +1,30 @@
+name: 'LEGO automerge'
+
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - Localization
+
+jobs:
+  worker:
+    runs-on: ubuntu-latest
+
+    if: github.actor == 'csigs'
+    steps:
+      - uses: actions-ecosystem/action-add-labels@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          labels: enhancement
```

Comment:
Could it make sense to add some specific label for this?

Reply:
In theory, we can add a separate label for localization PRs. But it will also require updating the label checker script since currently, it is checking a specific list of labels. If it is ok, I will add a specific label for such PRs and will update the label checker script

Reply:
@EzzhevNikita got it thanks! Let's leave this as is at the moment